src/pages/index.js

```diff
@@ -179,13 +225,23 @@ function getCardElement({ link, name }) {
   return cardElement;
 }
 
-function loadInitialCards() {
-  initialCards.forEach((cardData) => {
-    const cardEl = getCardElement(cardData);
-    cardsList.append(cardEl);
-  });
+function initialFetch() {
```

Comment:
Function names should start with a **verb** in infinitive (there is such a point in the checklist). It’s common in coding to do it so that functions could `speak` for themselves about what they do.


`getInitialData`